input/fsh/Profiles clinical datapoints/TumorStaging.fsh

```diff
@@ -0,0 +1,184 @@
+// CLINICAL TUMOR STAGE
+Profile:  ClinicalTNMTumorStage
+Id: clinical-tnm-tumor-stage
+Parent: Observation
+Title: "Clinical TNM tumor stage"
+Description: "Represents the clinical stage of the tumor (per AJCC 8th Ed.)"
+* insert PublicationProfileRuleset
+* code = SCT#385356007 "Tumor stage finding"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from TNMPrimaryTumorVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: ClinicalTNMTumorStagePatient147
+InstanceOf: ClinicalTNMTumorStage
+Description: "Example showing TNM staging for tumor stage"
+* status = #final "final"
+* subject = Reference(BreastCancerPatient147)
+* valueCodeableConcept = AJCC#cTis
+* method = SCT#897275008 "American Joint Commission on Cancer, Cancer Staging Manual, 8th edition neoplasm staging system (tumor staging)"
+
+Mapping: ClinicalTNMTumorStageToICHOM
+Source:	ClinicalTNMTumorStage
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: ClinicalTNMTumorStageMapping
+Title: "Clinical TNM tumor stage to ICHOM set"
+Description: "Mapping of the clinical TNM tumor stage to the ICHOM breast cancer PCOM set." 	
+* valueCodeableConcept -> "Clinical tumor stage"
+
+// PATHOLOGICAL TUMOR STAGE
+Profile:  PathologicalTNMTumorStage
+Id: pathological-tnm-tumor-stage
+Parent: Observation
+Title: "Pathological TNM tumor stage"
+Description: "Represents the pathological stage of the tumor (per AJCC 8th Ed.)"
+* insert PublicationProfileRuleset
+* code = SCT#385356007 "Tumor stage finding"
+* subject only Reference(BreastCancerPatient)
+* value[x] only CodeableConcept 
+* value[x] from TNMPrimaryTumorVS (preferred)
+* value[x] MS
+* effectiveDateTime MS
+
+Instance: PathologicalTNMTumorStagePatient147
```

Comment:
Shouldn't it also include the code and the effectiveDateTime? In my opinion it is important to provide complete examples.

code = SCT#385356007 "Tumor stage finding"